doc/release-notes/5.3-release-notes.md

```diff
@@ -132,27 +132,37 @@ Please configure your connection details, replacing all the `${DB_...}`.
 - `<payara install path>/bin/asadmin create-password-alias --passwordfile /tmp/password.txt dataverse.db.password`
 - `rm /tmp/password.txt`
 
-4\. Reset the EJB timer database back to default:
+4\. Replace relevant portions of domain.xml
+
+- In domain.xml, replace 
+ 
+- `<jdbc-connection-pool datasource-classname="org.apache.derby.jdbc.EmbeddedXADataSource" name="__TimerPool" res-type="javax.sql.XADataSource"> <property name="databaseName" value="${com.sun.aas.instanceRoot}/lib/databases/ejbtimer"></property><property name="connectionAttributes" value=";create=true"></property></jdbc-connection-pool>`
+
+with
+
+- `<jdbc-connection-pool datasource-classname="org.h2.jdbcx.JdbcDataSource" name="__TimerPool" res-type="javax.sql.XADataSource"> <property name="URL" value="jdbc:h2:${com.sun.aas.instanceRoot}/lib/databases/ejbtimer;AUTO_SERVER=TRUE"></property> </jdbc-connection-pool>`
```

Comment:
In the [dataverse-community thread](https://groups.google.com/g/dataverse-community/c/dR8KmFVFB6o/m/4j5u6DWLBAAJ) @jmjamison mentioned deleting the directory targeted by the property.

This might be necessary due to the change from Derby to H2. Not sure if the directory is re-created by Payara though.

Reply:
Thanks @poikilotherm - I'll discuss with @scolapasta and get the right content in here.

Reply:
LGTM